requirements-test.txt

```diff
@@ -3,3 +3,6 @@
 -r requirements.txt
 
 -e .[tests]
+
+# TODO: using the latest proposal version; remove that after it has been merged upstream
+-e git+https://github.com/palazzem/opentracing-python.git@palazzem/scope-manager#egg=opentracing-1.3.0
```

Comment:
I think you should update this requirement with your branch instead of mine, so that the CI can run properly.

Reply:
Totally oversaw this, will update 👍